### DISCUSSION TITLE
bucket shuffle join 和其他join相比是否消耗更多的FE资源

### DISCUSSION THREAD

**Discussion by yjant** [apache/doris#6857] [A - General / Q&A]
集群升级到0.14 后，原本可以运行的join查询，报错，
ERROR 1064 (HY000): failed to send brpc batch, error=Host is down, error_text=[E112]Not connected to 0.0.0.0:0 yet, server_id=8589935239 [R1][E112]Not connected to 0.0.0.0:0 yet, server_id=8589935239 [R2][E112]Not connected to 0.0.0.0:0 yet, server_id=8589935239 [R3][E112]Not connected to 0.0.0.0:0 yet, server_id=8589935239, client:  
但是实际上服务是正常运行的， 关闭bucket shuffle join 之后，又可以正常执行sql，
所以想请问下，是否是查询更消耗FE资源导致FE资源不够引起的。

**Comment by morningman** [comment]
This is bug of bucket shuffle join. You can
1. set enable_bucket_shuffle_join=false to void this error.
2. Upgrade to PALO-0.14.13.1, palo.baidu.com
